server/implementation-cdi/src/main/java/io/smallrye/graphql/cdi/validation/ValidationService.java

```diff
@@ -39,4 +40,13 @@ public void beforeInvoke(InvokeInfo invokeInfo) throws Exception {
     public String getConfigKey() {
         return ConfigKey.ENABLE_VALIDATION;
     }
+
+    private ValidatorFactory getValidatorFactory() {
+        try {
+            ValidatorFactory validatorFactory = CDI.current().select(ValidatorFactory.class).get();
```

Comment:
We discussed this in the meeting - could we use the CDI Lookup service to get the ValidatorFactory bean?